### DISCUSSION TITLE
Can we add ability the to unique on matches?

### DISCUSSION THREAD

**Discussion by apowis** [BurntSushi/ripgrep#2134] [General] [answered]
Feel like this could be a useful feature. What are other people's thoughts on this?

**Comment by BurntSushi** [comment, marked as answer]
What are you asking for exactly? As stated, this is incomprehensible to me. Consider providing some examples of the feature you're asking for in action.

**Reply by bdmorin** [reply]
I think the OP is saying build in "uniq" to ripgrep. So instead of piping the output to say `rg match thunder.out | uniq` have a switch that filters in ripgrep.

**Reply by BurntSushi** [reply]
Ah yeah I think I'm going to pass on that. Thanks for clarifying.
